res/values-ro/strings.xml

```diff
@@ -596,7 +596,7 @@
     <string name="PRIZES">"PREMII"</string>
 
     <string name="Enable_Custom_Skin">"Activează Skin-ul Personalizat"</string>
-    <string name="Use_Custom_Skin">"Skin-ul\nPersonalizat"</string>
+    <string name="Use_Custom_Skin">"Folosește Skin-ul Personalizat"</string>
```

Comment:
Please return the `\n` here between "Skin-ul" and "Personalizat"

Reply:
Ok sorry

Reply:
I didn't know

Reply:
When you're done with the changes, please mark this as resolved ^^

Reply:
It's solved